app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ReviewsController < ApplicationController
+  before_action :set_flat, only: %i[new create]
+
+  def new
+    @flat = Flat.find(params[:flat_id])
```

Comment:
if you do set_flat do you need this line?

Reply:
It is no longer needed